internal/postprocessor/main.go

```diff
@@ -193,15 +210,42 @@ func (c *config) RegenSnippets() error {
 	return nil
 }
 
-func (c *config) replaceAllForSnippets(googleCloudDir, snippetDir string) error {
-	return execv.ForEachMod(googleCloudDir, func(dir string) error {
+func (c *config) getConfs() []*generator.MicrogenConfig {
```

Comment:
This getter in more involved and selectively does things. Would you mind adding some godoc here.

Reply:
Done, I also changed the name to be more descriptive.